src/azul/__init__.py

```diff
@@ -1443,8 +1443,8 @@ def docker_registry(self) -> str:
         'docker.elastic.co/kibana/kibana-oss:7.10.2',
         'docker.io/clamav/clamav:1.1.0-1',
         'docker.io/cllunsford/aws-signing-proxy:0.2.2',
-        'docker.io/gitlab/gitlab-ce:16.0.3-ce.0',
-        'docker.io/gitlab/gitlab-runner:v16.0.1',
+        'docker.io/gitlab/gitlab-ce:16.0.6-ce.0',
```

Comment:
The latest minor release is 16.1 not 16.0. It was released 6/22. What's the reason you're not upgrading to that?

Additionally, a security release was made available today. We should include that as well.

Reply:
> The latest minor release is 16.1 not 16.0. It was released 6/22. What's the reason you're not upgrading to that?

There is an instruction in the [operator manual](https://github.com/DataBiosphere/azul/blob/develop/OPERATOR.rst#upgrading-gitlab-clamav) to update `gitlab-runner` to the same major & minor version that `gitlab-ce` is updated to. Since the latest available `v16` tag of `gitlab-runner` is v16.0.2, I have to keep `gitlab-ce` on the 16.0.x release.